docs/developer-guides/contributing.md

```diff
@@ -48,10 +48,10 @@ If you want to contribute benchmark results run by specified SuperBench version,
 
 #### Where to submit
 
-All the results are stored under `results` folder. The directory structure is as follows. Please create `<your-benchmark-folder>` to submit results.
+All the results are stored under [superbench-results](https://github.com/microsoft/superbench-results) repository. The directory structure is as follows. Please create `<your-benchmark-folder>` to submit results.
 
 ```
-└── results
+└── superbench-results
```

Comment:
remove `└──` and indent the following lines?

Reply:
sure. updated.